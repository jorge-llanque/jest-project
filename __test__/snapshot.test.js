import { getCharacter } from "../snapshot";
import rick from '../rick.json';

describe('Instantaneas', () => {
    test('Snapshot', () => {
        expect(getCharacter(rick)).toMatchSnapshot();
    })
    
})
