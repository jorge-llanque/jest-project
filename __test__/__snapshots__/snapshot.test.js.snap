// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instantaneas Snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanchez",
  "species": "Human",
  "status": "Alive",
}
`;
